.bazelrc

```diff
@@ -158,8 +158,8 @@ build:remote-msan --config=rbe-toolchain-clang-libc++
 build:remote-msan --config=rbe-toolchain-msan
 
 # Docker sandbox
-# NOTE: Update this from https://github.com/envoyproxy/envoy-build-tools/blob/master/toolchains/rbe_toolchains_config.bzl#L7
-build:docker-sandbox --experimental_docker_image=envoyproxy/envoy-build-ubuntu@sha256:3788a87461f2b3dc8048ad0ce5df40438a56e0a8f1a4ab0f61b4ef0d8c11ff1f
+# NOTE: The default building image uses the latest version of envoy-build-ubuntu in dockerhub for multi-arch supported.
```

Comment:
We have to verify that the image for x86_64 has the same digest as the sha256 in https://github.com/envoyproxy/envoy-build-tools/blob/master/toolchains/rbe_toolchains_config.bzl#L7, otherwise RBE will be inconsistent.

Reply:
Hi, Lizan. Thanks for your comments. 
Could you please give me more information about it? I do not really understand what your meaning.

Reply:
The image at https://github.com/envoyproxy/envoy-build-tools/blob/master/toolchains/rbe_toolchains_config.bzl#L7 must be the same image here. (i.e. bazel toolchain was successfully generated for that image). The information from dockerhub lacks that check so we can't just use what is latest there.